### DISCUSSION TITLE
Is it compatible with nwidart/paravel-modules?

### DISCUSSION THREAD

**Discussion by taotecode** [Laravel-Lang/common#10] [Q&A]
Because my current project is using nwidart/paravel-modules
I wonder if you can translate the folder in its Modules/Demo/Resources/lang as well?

![image](https://user-images.githubusercontent.com/48904825/218106572-9b3e33f3-85b3-47df-9d9c-f6fffe83905d.png)

**Comment by andrey-helldar** [comment]
`Laravel Lang: Lang` project does not translate keys added by the developer. It contains a translation of the standard Laravel project keys.

The current implementation of the [publisher](https://publisher.laravel-lang.com/) saves them only in the main localizations folder: `./lang` or `./resources/lang`.

**Reply by andrey-helldar** [reply]
The publisher looks for translations in the folder returned by calling the `lang_path()`.
